app/cdap/components/PipelineTriggers/PayloadConfigModal/index.js

```diff
@@ -33,41 +32,24 @@ export default class PayloadConfigModal extends Component {
     isOpen: PropTypes.bool,
     triggeringPipelineInfo: PropTypes.object,
     triggeredPipelineInfo: PropTypes.string,
-    onClose: PropTypes.func,
-    onEnableSchedule: PropTypes.func,
+    onConfigureSchedule: PropTypes.func,
     disabled: PropTypes.bool,
     scheduleInfo: PropTypes.object,
     configureError: PropTypes.string,
     onToggle: PropTypes.func,
-    andTriggersEnabled: PropTypes.bool,
-  };
-
-  state = {
```

Comment:
Moved out the payloadConfig state and the internal button. The original code brings the issue that the payload modal button is not easily adjustable.